Formula/clickhouse-cpp.rb

```diff
@@ -0,0 +1,77 @@
+class ClickhouseCpp < Formula
+  desc "C++ client library for ClickHouse"
+  homepage "https://github.com/ClickHouse/clickhouse-cpp#readme"
+  url "https://github.com/ClickHouse/clickhouse-cpp.git",
+      tag:      "1.5.0",
+      revision: "1415b5936a2ac2f084850b09057e05fb5798b2f1"
```

Comment:
Can we use a tarball instead of a Git clone?

Reply:
The repo may eventually have submodules. GitHub approach of creating tarballs is strange, it doesn't include submodule content in them. So this is just to be on the safe side.

Reply:
Let's use a tarball for now. Git clones are slow, expensive, and not as easily cacheable as tarballs.

Reply:
Sure. Changed.